src/connectors/scenes/WalletListConnector.js

```diff
@@ -28,6 +28,10 @@ const mapStateToProps = (state: State) => {
   const isWalletFiatBalanceVisible = state.ui.settings.isWalletFiatBalanceVisible
   const defaultFiat = SETTINGS_SELECTORS.getDefaultFiat(state)
 
+  const supportedWalletTypes = SETTINGS_SELECTORS.getSupportedWalletTypes(state)
+  const supportedWalletTypesArray: Array<any> = Object.values(supportedWalletTypes)
```

Comment:
This seems redundant. `supportedWalletTypes` is already an array of walletType objects. What's the `Object.values()` call for?

Reply:
I "typeof" it before and it shows it is an object with keyed numbers somehow. I was confused also at first

Reply:
typeof for an Array is an object with keyed numbers. Please remove Object.values()

Reply:
fixed